src/components/pages/Offer/Offer/Stocks/StockItem/StockItem.jsx

```diff
@@ -118,39 +120,87 @@ const StockItem = ({
   const isStockEditable = isEvent ? isEventStockEditable : isThingStockEditable
   const isStockDeletable = isEvent ? stock.isEventDeletable : !isOfferSynchronized
 
-  const saveChanges = useCallback(() => {
-    const payload = {
-      id: stock.id,
-      price: price ? price : 0,
-      quantity: totalQuantity ? totalQuantity : null,
+  useEffect(() => {
+    const errorMessages = Object.values(formErrors)
+    if (errorMessages.length) {
+      notifyError(Object.values(errorMessages))
     }
-    const thingPayload = {
-      ...payload,
-      bookingLimitDatetime: getBookingLimitDatetimeForThing(),
+  }, [formErrors, notifyError])
+
+  const isValid = useCallback(() => {
```

Comment:
```suggestion
  const isCurrentStockValid = useCallback(() => {
```
Le nom isValid m'irait bien si c'était une méthode d'un objet de type Stock où on pourrait faire quelque chose du style `Stock.isValid()` mais comme là on appelle cette fonction telle quelle sans lui passer d'argument puisqu'elle s'appuie sur l'était actuel du stock, j'aime bien donner un peu plus de contexte.

Reply:
On est dans le StockItem, il me semble que isValid est assez explicite, car il valid le StockItem.

Reply:
C'est juste que j'ai dû aller voir l'implémentation pour comprendre ce que validait cette fonction que je ne connaissais pas.